doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -465,3 +466,23 @@ complies with the CNCF Conformance test for the package managers to use in the
 lifecycle management of the applications they manage. The Reference Architecture
 does not recommend the usage of a Kubernetes Application package manager with a
 server side component installed to the Kubernetes Cluster (e.g.: Tiller).
+
+## 3.3 CaaS Manager - Cluster Lifecycle Management
```

Comment:
This should not be in the scope of RA2.

Reply:
Add a note that further specification of this is out of scope for Kali.

Reply:
added